### PR TITLE
Small style updates

### LINKED ISSUE #346: Long Address Label broken in Address Details overlay
image

### PR DESCRIPTION
**Pull request by nibhar**
According to @julianbauer's feedback:
 - Updates the login-file success transition. The text does now fade in with a short delay and does no longer appear instantaneous.
 - File export success screen close icon now has the correct styling.
 - Centers the label in the AddressInfo overlay.
 - Reduces the opacity of the blurred Login File.
 - fixes a bug that made the recovery words not scrollable on safari
 - fixes the display of the different states in validate words

Resolves #346.

### CODE REVIEW

File: src/request/export/Export.js [hunk 1]
```diff
@@ -85,7 +85,11 @@ class Export {
         if (this._request.fileOnly) {
             this._resolve(this.exported);
         } else {
+            this._fileSuccessPage.classList.add('display-flex');
+            // eslint-disable-next-line no-unused-expressions
+            this._fileSuccessPage.offsetHeight;
```
Comment (sisou) [inline src/request/export/Export.js]:
```suggestion
            this._fileSuccessPage.offsetHeight; // eslint-disable-line no-unused-expressions
```

File: src/request/export/Export.css [hunk 2]
```diff
@@ -186,6 +186,15 @@ ul.nq-list li > .nq-icon {
     background-color: rgb( 255, 255, 255, .0); /* transparent white */
 }
 
+.page#login-file-success {
+    opacity: 0;
+    transition: opacity .3s .1s ease;
```
Comment (danimoh) [inline src/request/export/Export.css]: Are you sure it's supposed to be `ease`? :D

Comment (nibhar) [reply src/request/export/Export.css]: Likely not, but that is why @julianbauer is also a reviewer here ;)

Comment (julianbauer) [reply src/request/export/Export.css]: For opacity ease is alright :)

**Pull request merged**